### ISSUE TITLE
UX Jackob's Law correction.

### ISSUE THREAD

**Issue by shelvaldes** [akiraux/Akira#565]
## Expected Behavior
Layers should be on the left and control tool option on the right. 

## Current Behavior
I saw the screenshots on the readme, they are the opposite

## Possible Solution
While this could seem unimportant, i'ts violating the Jakob's Law of UX, as any other UX design software already existing has this layout. Practically any UX designer is already used to the panels placed this way.

**Comment by Alecaddd** [comment]
> While this could seem unimportant, i'ts violating the Jakob's Law of UX, as any other UX design software already existing has this layout. Practically any UX designer is already used to the panels placed this way.

Not true.
Photoshopt, InDesign, Illustrator, Affinity Designer, and others have the layers panel tot he right by default.

> I saw the screenshots on the readme, they are the opposite

Don't open issues unless you used the app. Looking at a screenshot doesn't give you the necessary knowledge and you end up making wrong assumptions.

If you had used Akira, you would have found an option in the settings panel to change the order of the layout.

**Comment by shelvaldes** [comment]
Dude, I'm trying to help here. By the way, Photoshop, Illustrator, InDesign and Afinity designer are NOT UI design software, as they have their particular specialities. Sketch, Adobe XD and Figma are. Sorry for not having installed and tryed the software (yet), I promise I will. I'm a UX designer myself and I'm interested. But I didn't want to forget it cause I really wanted to, as I said, help. That's the spirit of FS, I guess. You may know by now it's not uncommon that Free software just ignores UX laws or designer's pipeline, or designers at all hahaha, so I made some assumptions. I'm glad the feature is already covered thoug. You didn't need to be rude, at least not with me, get a plant, or a rubber duck.

**Comment by Alecaddd** [comment]
Lol, I wasn't rude, I simply answered to your points.
I'm not mad or anything, I'm just very direct, sorry if I gave a sense of being rude, it wasn't my intention.

There was a discussion a while ago where I explained and analyzed the reason why the layers panel is on the right by default.
You can check it here if you want: https://github.com/akiraux/Akira/issues/79

But yeah, the app allows to change the order of the panels, and I'm planning to add more customization in the future because, even tho good defaults are "good", different users have different needs and workflows, so making Akira capable of adapting to those needs is one of my many goals.

Thanks for your interest in Akira.
